modin/core/dataframe/base/dataframe/dataframe.py

```diff
@@ -269,7 +276,7 @@ def reduce(
             The axis to perform the reduce over.
         function : callable(row|col) -> single value
             The reduce function to apply to each column.
-        dtypes : str, optional
+        dtypes : pandas.Series, optional
```

Comment:
Is scalar type not accepted here?

Reply:
Scalar types are not accepted here. The control flow here is `reduce` -> `_compute_tree_reduce_metadata` -> dataframe constructor, which directly sets `self.dtypes` on the new dataframe. In contrast, `map` is written to duplicate scalar datatypes into a series [here](https://github.com/modin-project/modin/blob/b6a460dd8707cbc87a2038ba251a72d8aa9d8eab/modin/core/dataframe/pandas/dataframe/dataframe.py#L1739).